test/integration/undo-redo/index.spec.ts

```diff
@@ -42,42 +41,3 @@ test('test basic undo and redo', async ({ page, browserName, api }) => {
   // Redo should bring back text field
   await expect(canvasInputLocator).toHaveCount(3);
 });
-
-test('test batching quick actions into single undo entry', async ({ page, browserName, api }) => {
```

Comment:
This spec should still be relevant, specifically for text field changes

Reply:
We can have something more specific once we have specific logic for text inputs?
Anyway I won't merge this until i have a test that does the same that this one did.

Reply:
This test should work fine after all, I had the impression that it was testing more general changes but I was wrong!